.vscode/launch.json

```diff
@@ -25,6 +25,12 @@
       "presentation": {
         "hidden": true
       }
+    },
+    {
+      "type": "lldb",
+      "request": "attach",
+      "name": "Debug Electron Renderer",
```

Comment:
Since this is native debugging, should it say "Debug native code" instead?

Reply:
Yeh it was copied across from another project, so this can be changed